.github/pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).
+**Contributions**: We are not currently taking public contributions - see our [contributions](../blob/master/.github/CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).
```

Comment:
relative link to blob found, did you mean `./` instead?

Reply:
nope, because this navigates from 

{repo}/pull/

so up a dir then down blob/master to the file

Reply:
same change is being done with starter project and blank project templates